tasks/main.yml

```diff
@@ -62,39 +61,46 @@
   when: cloned_mailman_bundler.changed
 
 - name: Setup a virtualenv for mailman
-  command: su - {{mailman_user}} -c "virtualenv {{mailman_path}}/venv" creates="{{mailman_path}}/venv"
+  command: su - {{mailman_user}} -c "virtualenv {{mailman_path}}/venv"
+  args:
+    creates: "{{mailman_path}}/venv"
```

Comment:
It is better to use creates in a separate line here since it is more clear what the command that is being run is.